spring-integration-core/src/main/java/org/springframework/integration/dsl/BaseIntegrationFlowDefinition.java

```diff
@@ -179,6 +180,21 @@ protected MessageChannel getCurrentMessageChannel() {
 		return this.currentMessageChannel;
 	}
 
+	/**
+	 * Returns the current channel if it is an {@link InterceptableChannel}, otherwise registers a new implicit
+	 * {@link DirectChannel} in the flow and returns that one.
+	 * @return the current channel after the operation
+	 */
+	protected MessageChannel makeCurrentMessageChannelInterceptable() {
```

Comment:
I think we need to return exactly an `InterceptableChannel` from here.
And let's call it like `currentInterceptableChannel()`! A "make" is confusing a little bit...